src/core/instance/state.js

```diff
@@ -292,7 +292,7 @@ function initWatch (vm: Component, watch: Object) {
 
 function createWatcher (
   vm: Component,
-  keyOrFn: string | Function,
+  expOrFn: string | Function,
```

Comment:
It's key path or function, IMO `keyOrFn` is better :)

Reply:
but it's using 
```
return createWatcher(vm, expOrFn, cb, options)
```
should we change `expOrFn` to `keyOrFn`, I just get confused when reading the code, exactly the same thing but using different name.

Reply:
Oops...you are right, it is `expOrFn` indeed.😂